include/deal.II/numerics/data_out_dof_data.templates.h

```diff
@@ -103,30 +103,43 @@ namespace internal
       unsigned int n_q_points = 0;
       if (use_face_values == false)
         {
-          const bool do_simplex =
+          const bool do_hex =
             std::any_of(finite_elements.begin(),
```

Comment:
Shouldn't this now be `std::all_of`?

Reply:
As a matter of fact, I now realize that you may be using this variable when asking whether you need to set something up. In that case, `std::any_of` is correct, but the variable name is not giving that away. Maybe `needs_hypercube_setup`?